src/Containers/Popup.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import { LabelLayoutProps } from '@Layouts';
+
+export interface PopupProps extends LabelLayoutProps {
+    left?: number;
+    top?: number;
+    width?: number;
+    height?: number;
+    popup?: boolean;
```

Comment:
does this do 'popup' prop do anything?

Reply:
It's a boolean saying whether component should have popup or not